riscv-page-tables/src/page_table.rs

```diff
@@ -624,7 +632,6 @@ impl<T: PagingMode> PageTableInner<T> {
 
 /// A paging hierarchy for a given addressing type.
 /// This is used for HS and HU paging modes such as Sv48.
-///
 /// TODO: Support non-4k page sizes.
```

Comment:
Can this TODO be removed or there's still some non-4k pages support to be added?